build/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3
```

Comment:
alpine is not needed. you can just have a `from scratch` docker image 

The `file` can then just copied from the output target during the built